src/picongpu/submit/submitAction.sh

```diff
@@ -24,6 +24,10 @@ mkdir picongpu
 cp -r $TBG_projectPath/bin picongpu
 cp -r $TBG_projectPath/include picongpu
 cp -r $TBG_projectPath/submit picongpu
+if [ -d $TBG_projectPath/pymodules ]
+then
+  cp -r $TBG_projectPath/pymodules picongpu
```

Comment:
Had an offline talk with @ax3l about this. The directories `include` and `submit` will be in the `picongpu` directory as well. Before, they are in their respective `examples/<MyExample>/` directory - as is the `pymodules` directory. Therefore it makes sense to treat all of those directories the same. Especially since the analysis tools in `pymodules` will most-likely be example-specific.